api/src/test/java/org/openmrs/api/PersonServiceTest.java

```diff
@@ -1730,6 +1743,27 @@ public void voidRelationship_shouldVoidRelationshipAndSetVoidedByToGivenUserIfGi
 		assertEquals(voidedRelationship.getVoidedBy(), user);
 	}
 	
+	@Test
+	public void voidRelationship_shouldVoidRelationshipAndSetVoidedByToGivenUserIfGivenRelationshipHasNullVoidedBy() {
+		
+		Relationship relationship = personService.getRelationship(1);
+		assertFalse("We need an unvoided relationship to test the method", relationship.getVoided());
+		String voidReason = "Something";
+		
+		// TODO - voiding is done by the BaseVoidHandler called via AOP before voidRelationship
```

Comment:
its bad practice to leave TODOs in code, if you create a new test, dont just copy an existing with a TODO. it will never be deleted as you can see in this test class.

Reply:
@teleivo ok I will remove them. Thanks